app/models/filesystem.rb

```diff
@@ -120,6 +120,15 @@ def contents_displayable?
     !mime_type.binary?
   end
 
+  def displayable_contents
+    if contents_displayable? &&
+       ((contents[0].ord == 255 && contents[1].ord == 254) || (contents[0].ord == 254 && contents[1].ord == 255))
```

Comment:
Can this be made less complicated to read with something like the following

```ruby
contents[0, 2] == "\xFF\xFE" || contents[0, 2] == "\xFE\xFF" # UTF_16LE and UTF_16BE BOMs
```

or just use constants instead of a comment?

```ruby
UTF_16BE_BOM = "\xFE\xFF"
UTF_16LE_BOM = "\xFF\xFE"
# ...
contents[0, 2] == UTF_16BE_BOM || contents[0, 2] == UTF_16LE_BOM
```